crmsh/bootstrap.py

```diff
@@ -1362,38 +1362,44 @@ def init_sbd():
             if not confirm("SBD is already configured to use %s - overwrite?" % (configured_dev)):
                 return
 
-        dev = ""
         dev_looks_sane = False
         while not dev_looks_sane:
```

Comment:
Now I see, what is this `dev_looks_sane` loop used for?
The loop finishes always in the 1st iteration, there is no way that goes to 2nd round.

If this is the case, we could just remove the loop for code simplicity

Reply:
`dev_looks_sane` is useful.
When 
1. input dev is not a block device, 
2. user answer `no` for `Are you sure you wish to use this device`

`dev_looks_sane` is False, then prompt the string to ask user input again.

Reply:
Ok, perfect! Now I get it! Thanks